src/components/RichTextEditor/RichTextEditor.css

```diff
@@ -9,14 +9,25 @@
   --editor-container-width: 100%;
 }
 
+.text-editor-container {
+  display: flex;
+  flex-direction: var(--rich-text-editor-flex-direction, column);
+}
+
 .is-active {
   background: var(--button-background-color-is-active);
   color: var(--button-color-is-active);
 }
 
 .menu-bar-container {
+  align-items: var(--menu-bar-align-items, flex-start);
+  background-color: var(--menu-bar-background-color, --structure-color-white);
```

Comment:
@viniciusaa can you please a correction in this line. The secondary line should have the `var` keyword. It should be written as: `var(--menu-bar-background-color, var(--structure-color-white));`